app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'pry-byebug'
+
+NUMBER_OF_SHOWS = 10
+class ShowsController < ApplicationController
+  def index
+    @shows = CSV.parse(File.read('lib/netflix_titles.csv'), headers: true).first(NUMBER_OF_SHOWS)
```

Comment:
```suggestion
    @shows = CSV.read('lib/netflix_titles.csv', headers: true).first(NUMBER_OF_SHOWS)
```
Optional: Avoid using `File` class needlessly

Reply:
Done, and also added a fixture